src/otx/core/model/module/base.py

```diff
@@ -135,12 +136,32 @@ def state_dict(self) -> dict[str, Any]:
         state_dict["meta_info"] = self.meta_info
         return state_dict
 
-    def load_state_dict(self, state_dict: dict[str, Any], *args, **kwargs) -> None:
+    def _load_from_prev_otx_ckpt(self, state_dict: dict[str, Any]) -> dict[str, Any]:
+        """Attach the model.model prefix to load without problem."""
+        msg = "Trying to load the model checkpoint created by OTX 1.X. it will be converted to OTX2.0 format."
+        warnings.warn(msg, stacklevel=1)
+        for key in list(state_dict.keys()):
+            value = state_dict.pop(key)
+            new_key = "model.model." + key
+            state_dict[new_key] = value
+        return state_dict
```

Comment:
Is really this function able to be used in common broadly? If not, please consider converting this to an abstract function. I'm feeling that this PR introduces so many complicate code levels compared to what we really want. It needs to downweighted.
```suggestion
    def _load_from_otx_1_x_ckpt(self, state_dict: dict[str, Any]) -> dict[str, Any]:
        """Attach the model.model prefix to load without problem."""
        msg = "Trying to load the model checkpoint created by OTX 1.X. But, there is no model checkpoint conversion implementation for this model."
        raise NotImplementError(msg)
```

p.s. Consider renaming it to `_load_from_otx_1_x_ckpt` to make it more specific.

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/2828/commits/7fedf31302bc0b4143d1e32dd99ff70237978cf7